Apps/Examples/Examples/All Examples/CustomAnchoredSymbolExample.swift

```diff
@@ -0,0 +1,321 @@
+                                id: "imageLeftHanded",
+                                sdf: false,
+                                stretchX: stretchX,
+                                stretchY: stretchY,
+                                content: imageContent)
+
+            try? style.addImage(image.tint(highlightTintColor),
+                                id: "imageLeftHanded-Highlighted",
+                                sdf: false,
+                                stretchX: stretchX,
+                                stretchY: stretchY,
+                                content: imageContent)
+        }
+    }
+
+    private func addFeatures() -> FeatureCollection {
+
+        var features = [Turf.Feature]()
+
+        let featureList = [
+            DebugFeature(coordinate: CLLocationCoordinate2DMake(40.714203, -74.006314), highlighted: false, sortOrder: 0, tailPosition: .left, label: "Chambers & Broadway - Lefthand Stem", imageName: "imageLeftHanded"),
+            DebugFeature(coordinate: CLLocationCoordinate2DMake(40.707918, -74.006008), highlighted: false, sortOrder: 0, tailPosition: .right, label: "Cliff & John - Righthand Stem", imageName: "imageRightHanded"),
+            DebugFeature(coordinate: CLLocationCoordinate2DMake(40.716281, -74.004526), highlighted: true, sortOrder: 1, tailPosition: .right, label: "Broadway & Worth - Right Highlighted", imageName: "imageRightHanded-Highlighted"),
+            DebugFeature(coordinate: CLLocationCoordinate2DMake(40.710194, -74.004248), highlighted: true, sortOrder: 1, tailPosition: .left, label: "Spruce & Gold - Left Highlighted", imageName: "imageLeftHanded-Highlighted"),
+            DebugFeature(coordinate: CLLocationCoordinate2DMake(40.7128, -74.0060), highlighted: true, sortOrder: 2, tailPosition: .center, label: "City Hall - Centered Highlighted", imageName: "imageCentered-Highlighted"),
+            DebugFeature(coordinate: CLLocationCoordinate2DMake(40.711427, -74.008614), highlighted: false, sortOrder: 3, tailPosition: .center, label: "Broadway & Vesey - Centered Stem", imageName: "imageCentered")
+        ]
+
+        for (index, feature) in featureList.enumerated() {
+            let point = Turf.Point(feature.coordinate)
+            var pointFeature = Feature(geometry: .point(point))
+
+            // Set the feature attributes which will be used in styling the symbol style layer.
+            pointFeature.properties = ["highlighted": feature.highlighted,
+                                       "tailPosition": feature.tailPosition.rawValue,
+                                       "text": feature.label,
+                                       "imageName": feature.imageName,
+                                       "sortOrder": feature.highlighted == true ? index : -index]
+
+            features.append(pointFeature)
+        }
+
+        return FeatureCollection(features: features)
+    }
+
+    private func addSymbolLayer(features: FeatureCollection) {
```

Comment:
nit: features implies [Feature] which is only 1 aspect of a [FeatureCollection](https://github.com/mapbox/turf-swift/blob/main/Sources/Turf/FeatureCollection.swift).

```suggestion
    private func addSymbolLayer(with featureCollection: FeatureCollection) {
```